src/nvim/ex_cmds2.c

```diff
@@ -2420,7 +2420,8 @@ void ex_compiler(exarg_T *eap)
 
   if (*eap->arg == NUL) {
     // List all compiler scripts.
-    do_cmdline_cmd("echo globpath(&rtp, 'compiler/*.vim')");  // NOLINT
+    // [vl][iu][ma] = lua | vim
+    do_cmdline_cmd("echo globpath(&rtp, 'compiler/*.[vl][iu][ma]')");  // NOLINT
```

Comment:
won't this also match `lim` or other random combinations?

Reply:
It actually will. I'll change that

Reply:
resolved